fixtures/stairville/af-150-dmx.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "AF-150 DMX",
```

Comment:
Maybe drop "DMX", for consistency with "AF-250"?  But it is part of the product name (for both).